pages/1.13/security/oss/login/local-user-login/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: layout.pug
+navigationTitle:  Local User Login
+title: Local User Login
+excerpt: Logging in to DC/OS as a local user
+menuWeight: 20
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# Login using the DC/OS CLI
+
+**Prerequisite:**
+- [DC/OS CLI](/1.13/cli/)
+
+Using the [DC/OS CLI](/1.13/cli/) one can log in as local DC/OS user by specifying the `dcos-users` login provider.
+
+1. To log in via the DC/OS CLI, replace `uid` and `password` in the following [auth login](/1.13/cli/command-reference/dcos-auth/dcos-auth-login/) command:
+
+```bash
+dcos auth login --provider=dcos-users --username=<uid> --password=<password>
+```
+
+2. Display the DC/OS authentication token by executing the following command:
+
+```bash
+dcos config show core.dcos_acs_token
+```
+
+3. Export the DC/OS Authentication token into environment for using it in other commands:
+```bash
+export TOKEN=$(dcos config show core.dcos_acs_token)
+```
+
+# Login using the IAM API
+
+Local users can log in using [Identity and Access Management (IAM) API](/1.13/security/oss/iam-api/).
+
+1. To log in local user accounts supply `<uid>` and `<password>` in the following command before executing it:
+
+```bash
+curl -X POST http://<host-ip>/acs/api/v1/auth/login -d '{"uid": "<uid>", "password": "<password>"}' -H 'Content-Type: application/json'
+```
+
+2. A DC/OS Authentication token will be returned in the HTTP response body similar to the one below:
```

Comment:
Rewrite `A DC/OS Authentication token will be returned in the HTTP response body similar to the one below:`
with `A DC/OS Authentication token will be returned in the HTTP response body similar to the below example:`